ang/civiawards/dashboard/directives/add-award-dashboard-action-button.controller.js

```diff
@@ -1,17 +1,18 @@
 (function (angular, getCrmUrl) {
   var module = angular.module('civiawards');
 
-  module.service('AddAwardDashboardActionButton', AddAwardDashboardActionButton);
+  module.controller('AddAwardDashboardActionButtonController', AddAwardDashboardActionButtonController);
 
   /**
-   * Handles the visibility and click event for the "Add Award" dashboard action button.
+   * "Add Award" dashboard action button controller.
    *
-   * @param {object} $location the location service.
-   * @param {object} $window the window object reference.
+   * @param {object} $scope the scope object
+   * @param {object} $location the location service
+   * @param {object} $window the window object reference
    */
-  function AddAwardDashboardActionButton ($location, $window) {
-    this.clickHandler = clickHandler;
-    this.isVisible = isVisible;
+  function AddAwardDashboardActionButtonController ($scope, $location, $window) {
+    $scope.clickHandler = clickHandler;
```

Comment:
since this is not a generic method any longer we should rename it to a more specific `redirectToNewAwardForm` or similar.